### PR TITLE
Scrollable job result

### PR DESCRIPTION
**Pull request by Mari-Hayashi**
Added scroll bars to the following pages:

1. Professor job dashboard
2. Student job search
3. Student job dashboard
scrollview

### CODE REVIEW

**Comment by johan1505** [comment]
> Added scroll bars to the following pages:
> 
> 1. Professor job dashboard
> 2. Student job search
> 3. Student job dashboard
>    scrollview

I thought we agreed we were also going to increase the number of jobs requested to the server? Will you add that to another pr?

**Comment by johan1505** [comment]
> Added scroll bars to the following pages:
> 
> 1. Professor job dashboard
> 2. Student job search
> 3. Student job dashboard
>    scrollview

There is some weird spacing problem when the number of jobs is very small

issue

File: src/client/src/Domains/Jobs/JobResults/index.tsx [hunk 1]
```diff
@@ -28,7 +28,9 @@ function JobResults({ jobs }: JobResultsProps) {
 
     return (
         <Grid container spacing={5} style={{ marginTop: 20 }}>
-            <Grid container item direction='column' spacing={3} xs={12} md={5}>
+            <Grid container item direction='row'
```
Comment (johan1505) [inline src/client/src/Domains/Jobs/JobResults/index.tsx]: I think the visual spacing bug comes from changing the direction to row.

**Review by johan1505 — CHANGES_REQUESTED** [review]

**Comment by johan1505** [comment]
> Added scroll bars to the following pages:
> 
> 1. Professor job dashboard
> 2. Student job search
> 3. Student job dashboard
>    scrollview

Now there is no spacing between the jobs
image

**Comment by Mari-Hayashi** [comment]
> > Added scroll bars to the following pages:
> > 
> > 1. Professor job dashboard
> > 2. Student job search
> > 3. Student job dashboard
> >    scrollview
> 
> Now there is no spacing between the jobs
> image

Yeah that's the only way I can do it. (I sent on slack.) Do you think it's fine, or do you want spacing?

**Comment by Mari-Hayashi** [comment]
> > > Added scroll bars to the following pages:
> > > 
> > > 1. Professor job dashboard
> > > 2. Student job search
> > > 3. Student job dashboard
> > >    scrollview
> > 
> > 
> > Now there is no spacing between the jobs
> > image
> 
> Yeah that's the only way I can do it. (I sent on slack.) Do you think it's fine, or do you want spacing?
Is this good?
scrollview2

scrollview2

**Comment by johan1505** [comment]
good

**Pull request merged**